tiingo/api.py

```diff
@@ -108,17 +108,21 @@ def list_tickers(self, assetType):
         raw_csv = get_buffer_from_zipfile(zipdata, 'supported_tickers.csv')
         reader = csv.DictReader(raw_csv)
 
+        if not len(assetTypes):
+            return [row for row in reader]
+
+        t = set(assetTypes)
```

Comment:
In general, we try not to have single letter variable names unless we're talking about coordinates (like x,y) or iteration variables (like i,j,k).  Could we call this something like `assetTypesSet`?

Reply:
I was going to do it without any proxy variables like `t` (which stands for `temp`). like this:
```
return [row for row in reader
                if row.get('assetType') in set(assetTypes)]
```
But in that case, the casting would be run for each row.
It is okay to name it something else though :)